examples/ogl_tutorials/tutorial03_matrices/src/maths.ads

```diff
@@ -0,0 +1,34 @@
+
+with Ada.Numerics;
+with Ada.Numerics.Generic_Elementary_Functions;
+
+with GL.Types;
+
+package Maths is
```

Comment:
I do not quite understand why this package exists. `+`, `-` and `*` are well-defined for both `GL.Types.Singles.Vector3` and `GL.Types.Singles.Matrix4`, as defined in `GL.Vectors` and `GL.Matrices` respectively. This example confuses users as it suggests you need to write your own implementation for these mathematical functions although they already exist. Please remove them or explain why you cannot use those from OpenGLAda.